packages/design-system/src/styles/components/_Table.scss

```diff
@@ -30,26 +30,32 @@ $table-stacked-header-max-width: 160px !default;
   th {
     border: 0;
     border-top: 1px solid;
+
+    /* autoprefixer grid: on */
     display: grid;
     // CSS grid doesn't support word-wrap. Use minmax as a workaround
     // https://css-tricks.com/css-grid-in-ie-debunking-common-ie-grid-misconceptions/#fit-content-is-not-ie-friendly-but
     grid-template-columns: minmax(0, $table-stacked-header-max-width) 1fr;
     position: relative;
 
-    // stylelint-disable selector-no-qualifying-type
+    /* stylelint-disable-next-line selector-no-qualifying-type */
     &[data-title]::before {
       content: attr(data-title);
-      // stylelint-enable selector-no-qualifying-type
-      flex: none;
       font-weight: bold;
+
+      /* autoprefixer grid: on */
+      grid-column: 1;
       max-width: $table-stacked-header-max-width;
-      // allow for spacing on right side of data-title column
       width: calc(100% - #{$spacer-2});
-      word-wrap: break-word;
     }
 
-    > * {
+    > span {
```

Comment:
I know you already covered this comment in the JSX file, but it would be helpful to add a comment here to explain that TableCell content is wrapped in a span for IE11 grid support